### PR TITLE
RD-4416: Provided ability to drilldown to the blueprintMarketplace page

### PR DESCRIPTION
**Pull request by NorbertAmartus**
[RD-4416](https://cloudifysource.atlassian.net/browse/RD-4416)

## Description

Changes consists of opening a `blueprintMarketplace` page upon clicking one of the `FirstUserJourneyButton`

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2320/)
(tests has been executed successfully, the `failure` label is coming from the coverage check - which was expected to fail)

## Documentation

Documentation will be updated with the usage of [this PR](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2150)
Synchronization should take place within this PR 🎸

### CODE REVIEW

File: test/cypress/integration/widgets/deployments_spec.ts [hunk 1]
```diff
@@ -101,15 +111,19 @@ describe('Deployments widget', () => {
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_spec.ts:111]: Is the name of the test case accurate? 

"when there's at least one deployment installed" -> "when there's no deployment installed"?

File: test/cypress/integration/widgets/deployments_spec.ts [hunk 2]
```diff
@@ -101,15 +111,19 @@ describe('Deployments widget', () => {
...
                     });
+                cy.contains('button', 'Yes').click();
 
-                cy.contains('Upload from Terraform').click();
-                cy.contains('Create blueprint from Terraform').should('be.visible');
+                cy.contains('Create new Deployment').click();
+                cy.contains('Blueprint Marketplace').should('be.visible');
+
+                // NOTE: Just so other test scenarios wouldn't be executed on the Blueprint Marketplace page
+                cy.go('back');
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_spec.ts:126]: That introduces dependency between tests. Maybe we can just have:
```
beforeEach(visitTestPage);
```
inside `describe('showFirstUserJourneyButtons option ...`?

File: test/cypress/integration/widgets/deployments_spec.ts (hunk 1 above)
Comment (NorbertAmartus) [reply test/cypress/integration/widgets/deployments_spec.ts:111]: Yeah, you're right! 😁 
It seems like I've mixed the test names 😅 

I've swapped them to resolve that issue 🚀

File: test/cypress/integration/widgets/deployments_spec.ts (hunk 2 above)
Comment (NorbertAmartus) [reply test/cypress/integration/widgets/deployments_spec.ts]: It'll still introduce dependency, when someone would omit the `should hide showFirstUserJourneyButtons view when there's at least one deployment installed` one, then the tests after that woul start to fail

But I know that it's some sort of edge case, so I'll implement the before mechanism, at it seems cleaner and more related to the cypress good practices 🎉 

Thanks for the suggestion 😎

File: test/cypress/integration/widgets/deployments_spec.ts [hunk 3]
```diff
@@ -77,6 +87,7 @@ describe('Deployments widget', () => {
 
         describe('showFirstUserJourneyButtons option and', () => {
             before(() => {
+                cy.visitTestPage();
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply test/cypress/integration/widgets/deployments_spec.ts:90]: Oh yeah, my mistake
I've put that into the before, run the tests and were happy that they were passing 😅 

Adjusting right away! :)

**Review by qooban — DISMISSED** [review]

**Comment by NorbertAmartus** [comment]
Documentation has been synchronized 🎉

**Review by qooban — APPROVED** [review]
